zerocloud/configparser.py

```diff
@@ -872,6 +872,7 @@ def __init__(self, id=None, name=None, exe=None, args=None, env=None,
         self.access = ''
         self.exe_name = exe_name
         self.data_in = False
+        self.location = location or None
```

Comment:
It's documented above. :)
I don't think adding "should not be empty string" is useful. Because it's not an error, and some people may want to add empty string as a way to say "no co-location this time"